src/server/requirements.txt

```diff
@@ -1,3 +1,4 @@
 flask~=2.2.3
 gunicorn~=20.0.4
-pyopenssl~=23.0.0
\ No newline at end of file
+pandas~=1.5.3
```

Comment:
To będzie używane po stronie serwera, czy tylko uczenia?

Reply:
The problem with this change is that there is a temporary file in `res/` directory, which uses `pandas`. There is no `requirements.txt` file for model training (because for some reason it wasn't provided).

Pylint on CI needs to install `pandas` in order to pass the checks, so it had to be added somewhere